bot/util/utilities.py

```diff
@@ -198,6 +268,64 @@ def generate_features_string(features):
     return str_features
 
 
+def get_text_channel(channel_id: str, guild: discord.Guild) -> discord.TextChannel:
+    """Parses a message id string and searches the text channel in the passed guild
+
+    Args:
+        channel_id (str): The id of the channel (might also be surrounded by '<#' and '>'
+        guild (discord.Guild): The guild in which the channel is searched.
+
+    Returns:
+        discord.Channel: The found channel.
+
+    Raises:
+        ValueError: If the chanel could not be found.
+    """
+    try:
+        if channel_id.startswith('<#'):
+            channel_id = channel_id[2:-1]
+        channel = guild.get_channel(int(channel_id))
+    except ValueError:
+        raise ValueError('Channel to post embed to could not be found. Use channel ID or channel name (linked #)')
+
+    if channel is None:
+        raise ValueError('Channel to post embed to could not be found. Use channel ID or channel name (linked #)')
+
+    return channel
+
+
+def is_pastebin_link(json_string: str) -> bool:
+    """Verifies if the string is a link to pastebin.com by checking if it contains 'pastebin.com' and does not contain
+    json specific symbols.
+
+    Args:
+        json_string (str): The string to be checked.
+
+    Returns:
+          bool: True if it is a link to pastebin.com, False if not.
+    """
+    return "pastebin.com" in json_string and '{' not in json_string and '}' not in json_string
```

Comment:
More pythonic approach and slightly shorter:
`return "pastebin.com" in json_string and not any(x in json_string for x in ("{", "}"))`